src/Calypso-SystemTools-Core/SycShowLocalSendersCommand.extension.st

```diff
@@ -0,0 +1,31 @@
+Extension { #name : 'SycShowLocalSendersCommand' }
+
+{ #category : '*Calypso-SystemTools-Core' }
+SycShowLocalSendersCommand class >> browserMenuOrder [
+	^ 2
+]
+
+{ #category : '*Calypso-SystemTools-Core' }
+SycShowLocalSendersCommand class >> methodMenuActivation [
+	<classAnnotation>
+
+	^ CmdContextMenuActivation
+		byRootGroupItemOrder: self browserMenuOrder 
+		for: ClyMethod asCalypsoItemContext
+]
+
+{ #category : '*Calypso-SystemTools-Core' }
+SycShowLocalSendersCommand class >> sourceCodeMenuActivation [
+	<classAnnotation>
+
+	^SycSourceCodeMenuActivation byRootGroupItemOrder: 1.2 for: ClyMethodSourceCodeContext
+]
+
+{ #category : '*Calypso-SystemTools-Core' }
+SycShowLocalSendersCommand class >> sourceCodeShortcutActivation [
+	<classAnnotation>
+
+	^ CmdShortcutActivation 
+		by: $v meta
```

Comment:
I wonder what impact it has. On mac cmd+v is a classic Paste from clipboard command.
So I guess such shortcut here will be fully ignored as it is intercepted inside internals of the text editor. Or the worse case it will break a copy paste behaviour